packages/database/supabase/migrations/20250619021503_correct_migrations.sql

```diff
@@ -0,0 +1,69 @@
+-- complete migration 20250610201801_create_access_table.sql
+
+revoke delete on table "public"."access_token" from "anon";
+
+revoke references on table "public"."access_token" from "anon";
+
+revoke trigger on table "public"."access_token" from "anon";
+
+revoke truncate on table "public"."access_token" from "anon";
+
+revoke update on table "public"."access_token" from "anon";
+
+-- rename in schema was not ported to migration 20250611180757_concept_upsert_functions.sql
+
+alter function "public".local_concept_to_db_concept(data concept_local_input) rename to _local_concept_to_db_concept;
+
+CREATE OR REPLACE FUNCTION public.upsert_concepts(v_space_id bigint, data jsonb)
```

Comment:
What error is this fixing?  Could you link the the source of the error?

Reply:
This is because I renamed the function to indicate it's private in the schema but forgot to do so in the migration. So in this migration I rename the private function, and I have to re-define the function that uses  it to use the new name. It is the only change to that function. Here I'm clearly migrating to align to the schema as source of truth.

Reply:
https://github.com/DiscourseGraphs/discourse-graph/pull/215/files#diff-22bac1fb1de78f55daaeedaaa408e1bd563554544a152bf0c82630d8675a69daR139